vapi/pci.vapi

```diff
@@ -0,0 +1,257 @@
+ */
+
+    public const int FILL_IDENT;
+    public const int FILL_IRQ;
+    public const int FILL_BASES;
+    public const int FILL_ROM_BASE;
+    public const int FILL_SIZES;
+    public const int FILL_CLASS;
+    public const int FILL_CAPS;
+    public const int FILL_EXT_CAPS;
+    public const int FILL_PHYS_SLOT;
+    public const int FILL_MODULE_ALIAS;
+    public const int FILL_LABEL;
+    public const int FILL_NUMA_NODE;
+    public const int FILL_IO_FLAGS;
+    public const int FILL_DT_NODE;      /* Device tree node */
+    public const int FILL_IOMMU_GROUP;
+    public const int FILL_BRIDGE_BASES;
+    public const int FILL_RESCAN;
+    [Version (since = "3.8.0")]
+    public const int FILL_CLASS_EXT;   /* prog_if and rev_id */
+    [Version (since = "3.8.0")]
+    public const int FILL_SUBSYS;   /* subsys_vendor_id and subsys_id */
+    public const int FILL_PARENT;
+    public const int FILL_DRIVER;   /* OS driver currently in use (string property) */
+
+    /*
+     * Conversion of PCI ID's to names (according to the pci.ids file)
+     *
+     * Call pci_lookup_name() to identify different types of ID's:
+     *
+     * VENDOR    (vendorID) -> vendor
+     * DEVICE    (vendorID, deviceID) -> device
+     * VENDOR | DEVICE   (vendorID, deviceID) -> combined vendor and device
+     * SUBSYSTEM | VENDOR  (subvendorID) -> subsystem vendor
+     * SUBSYSTEM | DEVICE  (vendorID, deviceID, subvendorID, subdevID) -> subsystem device
+     * SUBSYSTEM | VENDOR | DEVICE (vendorID, deviceID, subvendorID, subdevID) -> combined subsystem v+d
+     * SUBSYSTEM | ...   (-1, -1, subvendorID, subdevID) -> generic subsystem
+     * CLASS    (classID) -> class
+     * PROGIF    (classID, progif) -> programming interface
+     */
+    [CCode (cname = "pci_lookup_mode", cprefix = "PCI_LOOKUP_", has_type_id = false)]
+    enum LookupMode {
+        /** Vendor name (args: vendorID) */
+        VENDOR = 1,
+        /** Device name (args: vendorID, deviceID) */
+        DEVICE = 2,
+        /** Device class (args: classID) */
+        CLASS = 4,
+        SUBSYSTEM = 8,
+        /** Programming interface (args: classID, prog_if) */
+        PROGIF = 16,
+        /** Want only formatted numbers; default if access->numeric_ids is set */
+        NUMERIC = 0x10000,
+        /** Return NULL if not found in the database; default is to print numerically */
+        NO_NUMBERS = 0x20000,
+        /** Include both numbers and names */
+        MIXED = 0x40000,
+        /** Try to resolve unknown ID's by DNS */
+        NETWORK = 0x80000,
+        /** Do not consult local database */
+        SKIP_LOCAL = 0x100000,
+        /** Consult the local cache before using DNS */
+        CACHE = 0x200000,
+        /** Forget all previously cached entries, but still allow updating the cache */
+        REFRESH_CACHE = 0x400000,
+        /** Do not ask udev's hwdb */
+        NO_HWDB = 0x800000,
+    }
```

Comment:
You should use [Flags] for this: https://docs.vala.dev/developer-guides/bindings/writing-a-vapi-manually/04-00-recognizing-vala-semantics-in-c-code/04-02-enums-and-flags.html

Reply:
This enum now has `[Flags]` and values were removed.